api3-whitepaper.tex

```diff
@@ -211,7 +211,7 @@ \subsection{Decentralized APIs}
     \label{fig:solutions-comparison}
 \end{figure}
 
-This new definition of the problem implies that decentralized applications require specific Web API services to be delivered to the blockchain and this to be done in a fully decentralized, cost-efficient and secure way.
```

Comment:
The current version:
> The new definition implies
> 1. dApps need API integrations
> 2. These integrations to be done in a decentralized, cost-efficient and secure way

After the change:
> The new definition implies dApps need decentralized, cost-efficient and secure integrations

The current version tells that although 1 may exist (and it does, with any other oracle solution), 2 is also a requirement. It feels like that gets lost in the change.

Reply:
Ah yes I can see that, although then the active voice might help such that "applications require... services to be delivered to the blockchain and this to be done..." becomes "applications require delivery of... services to the blockchain and that this be done...". Otherwise, feel free to close.